docs/QUICK_START.md

```diff
@@ -0,0 +1,193 @@
+
+Carefully install your Tx module into the bay of your OpenTX or EdgeTX radio.  Some Tx modules can be fussy about pin alignment;  Don't use excessive force or you may break something.
+
+For the R9M Tx module, it is recommended to [Flash via OpenTx/EdgeTx](FRSKY_R9.md#flash-the-r9m-module-with-elrs-bootloader) which is a 2 step process:
+
+- First, [Flash the ELRS Bootloader](FRSKY_R9.md#flash-the-elrs-bootloader).
+- Next, use your OpenTx/EdgeTx radio to [Flash the mLRS Firmware](FRSKY_R9.md#flashupdate-the-mlrs-firmware).
+
+To flash ELRS receivers, you can use esptool or a Chrome or Chromium based browser as instructed [here](ELRS_RECEIVERS.md#flashing)
+
+[Flashing firmware](https://www.mateksys.com/?portfolio=mr24-30-tx#tab-id-3) to the MatekSys mLRS Tx modules is especially easy.
+
+Once the firmware is installed, you can confirm operation by observing the LED.
+- When either a receiver or Tx module is powered on by itself, a LED (usually red) should flash at 2 Hz (twice per second, on for 1/4 second then off for 1/4 second repeatedly).
+- When the Tx module and receiver are both powered on and bound, a LED (usually green) should flash more slowly at 1 Hz (once per second, on for 1/2 second then off for 1/2 second repeatedly) to indicate they are connected and exchanging radio messages.
+
+## 5. Install the mLRS LUA script and configure your mLRS system
+
+Follow the [instructions](LUA.md) and copy the correct mLRS LUA script to the SCRIPTS\TOOLS folder on your SD card and setup your RC radio external module.
+
+Start mLRS Configurator LUA script from the SYS->TOOLS menu and confirm you can talk to the Tx module.  You should see the TX module firmware version reported at the top of the screen.
+
+Use the mLRS LUA script to Confirm the receiver is connected and the firmware versions are recent enough to support the features you need.
+- If the radio link does not connect, follow the [Binding](BINDING.md) procedure.
+- It is HIGHLY recommended to use the same firmware version on both the receiver and Tx module. The latest released firmware is also recommended, especially for new installs.  If your Rx and Tx module firmware versions don't match or are very much out of date, return to the previous step and install the latest mLRS firmware release.
+
+Use the mLRS LUA script to configure your mLRS system
+- Leave your Rx powered on and connected while you complete the following steps.
+- Set the correct RF Band for your region (If using 900 MHz hardware).
+- You can also change the Bind Phrase if desired.
+- Set your desired Mode
+- If you didn't connect sbus or CRSF RC output from your receiver to your flight controller, enter the "Edit Rx" page and set Snd RcChannels to "rc override" (or "rc channels" for ArduPilot 4.6 with RC Channels support).  Return to the main page.  If your radio has a black and white screen, you will need to use the [CLI](CLI.md) for this.
```

Comment:
One can also modify the Lua to enable that setting - not sure if worth mentioning here though.

Reply:
I hope we can expand the B&W Lua.  (Just realized I keep Writing LUA.  I'll fix that.)

Reply:
I think best not to mention that here since it's not something a new user is likely to do.